script.js

```diff
@@ -1,41 +1,79 @@
 const playAgainButton = document.querySelector("#playagain");
-const h1 = document.querySelector("#heading");
+const head = document.querySelector("#heading");
 const yourScore = document.querySelector("#score");
 const blankStart = document.querySelector("#blank");
 const diceScore = document.querySelector("#dice");
 const rollButton = document.querySelector("#submit");
 
-
-let randomNumber = Math.floor(Math.random()*(7-1)+1);
 let totScore = 0;
 
-
+// if (randomNumber == 1){
+//     head.textContent= "You Lost"
+//     totScore = 0;
+//     playAgainButton.textContent = "Play Again";
+// }
 
 
 rollButton.addEventListener("click", () => {
+    let randomNumber = Math.ceil(Math.random()* 6 );
+    totScore += randomNumber;
+    yourScore.innerHTML = totScore;
     console.log(randomNumber);
-    totScore = totScore + randomNumber;
+    console.log(totScore);
+    diceScore.src = `img/dice${randomNumber}.png`;
```

Comment:
Nice, makes code much easier to read and also gives you much less to have to think about!

Reply:
Genuinely felt like a genius when I removed all the if lines and line 23 worked lol